### PR TITLE
Release 9.1.0

### PR DESCRIPTION
**Pull request by root-expert**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -624,13 +632,13 @@ Last release with Debian 8 support! After this 8.1.0 release we will do a 9.0.0
...
+## [0.2.1](https://github.com/voxpupuli/puppet-nodejs/tree/0.2.1) (2012-12-28)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-nodejs/compare/0.2.1...0.3.1)
+[Full Changelog](https://github.com/voxpupuli/puppet-nodejs/compare/0.3.1...0.2.1)
 
-## [0.2.1](https://github.com/voxpupuli/puppet-nodejs/tree/0.2.1) (2012-12-28)
+## [0.3.1](https://github.com/voxpupuli/puppet-nodejs/tree/0.3.1) (2012-12-28)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-nodejs/compare/0.2.0...0.2.1)
+[Full Changelog](https://github.com/voxpupuli/puppet-nodejs/compare/0.2.0...0.3.1)
```
[the remark this answers is not in the document]
Comment (smortex) [reply CHANGELOG.md:641]: Looks like 0.2.1 and 0.3.1 are the same commit and before 0.3.0.  Probably a tag failure…

Versions before 1.0.0 where not published to the forge, so let's just remove this buggy release…

**Review by bastelfreak — APPROVED** [review]

**Comment by root-expert** [comment]
Thanks @smortex, my changelog was still messed up for some reason :man_shrugging:

**Comment by smortex** [comment]
@root-expert The 0.3.1 tag was probably a mistake ;-)  I tried to fix the CentOS 7 failure in #453 but no luck so far :sob: If you have an idea…

**Comment by kenyon** [comment]
I'll do this release and then work on all the pull requests.

**Comment by evgeni** [comment]
mhh, this was released as 9.1.0 but contained a breaking change (https://github.com/voxpupuli/puppet-nodejs/pull/467)

**Comment by kenyon** [comment]
I should have regenerated the changelog myself before merging this, then I would have noticed that. :(

**Pull request merged**